web-api/src/storage/postgres.rs

```diff
@@ -216,41 +228,42 @@ impl Database {
                 .reset()
                 .push_tuple(ids)
                 .build()
-                .fetch_all(&mut tx)
+                .fetch_all(&mut *tx)
                 .await?
             {
                 existing_ids.push(DocumentId::from_row(&id)?);
             }
         }
-
-        tx.commit().await?;
-
         Ok(existing_ids)
     }
-}
 
-#[derive(FromRow)]
-struct QueriedCoi {
-    coi_id: Uuid,
-    is_positive: bool,
-    embedding: Vec<f32>,
-    /// The count is a `usize` stored as `i32` in database
-    view_count: i32,
-    /// The time is a `u64` stored as `i64` in database
-    view_time_ms: i64,
-    last_view: DateTime<Utc>,
-}
```

Comment:
I haven't touched this, the github merge algorithm just majorly messed up the diff :face_exhaling: